insights/parsers/sysconfig.py

```diff
@@ -506,3 +509,52 @@ class VirtWhoSysconfig(SysconfigOptions):
         'A TEST'
     """
     pass
+
+
+@parser(Specs.ifcfg_static_route)
+class IfCFGStaticRoute(LegacyItemAccess, Parser):
+    """
+    IfCFGStaticRoute is a parser for the static route network interface
+    definition files in ``/etc/sysconfig/network-scripts``.  These are
+    pulled into the network scripts using ``source``, so they are mainly
+    ``bash`` environment declarations of the form **KEY=value**.  These
+    are stored in the ``data`` property as a dictionary.  Quotes surrounding
+    the value
+
+    Because this parser reads multiple files, the interfaces are stored as a
+    list within the parser and need to be iterated through in order to find
+    specific interfaces.
+
+    Sample configuration from a static connection in file ``/etc/sysconfig/network-scripts/rute-test-net``::
+
+        ADDRESS0=10.65.223.0
+        NETMASK0=255.255.254.0
+        GATEWAY0=10.65.223.1
+
+    Examples:
+
+        >>> conn_info.static_route
+        'test-net'
+        >>> conn_info.data['ADDRESS0']
+        '10.65.223.0'
+        >>> sorted(conn_info.data.items())
+        [('ADDRESS0', '10.65.223.0'), ('GATEWAY0', '10.65.223.1'), ('NETMASK0', '255.255.254.0')]
+
+    Properties:
+        static_route (str): static route file name derived from file path
+        data (dict): all static route connection details
+    """
+    def __init__(self, context):
+        self.data = {}
+        self.static_route = context.path.split("network-scripts/route-", 1)[1]
```

Comment:
After changing its superclass to `SysconfigOptions`, the `__init__` and `parse_content` can be removed.

And the `self.static_route` could be:
```python
    @property
    def static_route(self):
        """(str): static route file name derived from file path"""
        return self.file_name.split("-", 1)[1]
```

Reply:
acknowledged, updated the code in the latest PR

Reply:
@xiangce Can we use "return self.file_name.split("route-", 1)[1]", so that we can only get route files, please suggest